vendor/brave-ios/objc-gen/objc-gen/Interface.swift

```diff
@@ -38,6 +38,7 @@ struct Interface: Hashable, Comparable {
   /// The generated output of a this @interface
   var generatedPublicInterface: String {
     return """
+    OBJC_EXPORT
```

Comment:
This file isn't even used anymore :) So this change can be removed. In fact, if you want you can completely delete objc-gen since its done by mojom now

Reply:
I think that should probably be a separate PR